case_specification.tex

```diff
@@ -0,0 +1,300 @@
+\subsection{Assumptions}
+
+\begin{frame}
+	\frametitle{Assumptions}
+	\begin{itemize}
+		\item fuel cycle facility parameters (throughput, availability)
+		\item compositions of fresh and spent fuel
+		\item material flow
+	\end{itemize}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Assumptions}
+	\begin{itemize}
+		\item SFR technology available for deployment in 2040.
+		\item Decay is not taken into account.
```

Comment:
Question: why not? Cyclus can handle decay.

Reply:
Originally I turned off decay because it significantly reduced the simulation time.. maybe I should do this again with decay on.